docs/crio.conf.5.md

```diff
@@ -36,11 +36,29 @@ The `crio` table supports the following options:
   CRIO state dir (default: "/var/run/containers/storage")
 
 **storage_driver**=""
-  CRIO storage driver (default is "devicemapper")
+  CRIO storage driver (default is "overlay")
 
 **storage_option**=[]
   CRIO storage driver option list (no default)
 
+  Values:
+
+	additionalimagestores=[]
+
+	Paths to additional container image stores. These are read/only and are usually stored on remote network shares.
+	storage_option=[ "overlay.additionaimagestores=/mnt/overlay" ]
```

Comment:
Should this be `overlay.imagestore=/mnt/overlay`?

Reply:
Yes fixed it.